src/wp-includes/blocks/index.php

```diff
@@ -12,6 +12,54 @@
 require BLOCKS_PATH . 'widget-group.php';
 require BLOCKS_PATH . 'require-dynamic-blocks.php';
 
+/**
+ * Registers core block types styles.
+ *
+ * @since 6.3.0
+ */
+function register_core_block_types_styles() {
```

Comment:
I think a better name would be `register_core_block_style_handles()`, because this function basically does what otherwise `register_block_style_handle()` would do, only in a more efficient way for core blocks.